package.json

```diff
@@ -9,17 +9,18 @@
     "lint": "npm run lint:standard && npm run lint:typescript",
     "lint:standard": "standard --verbose | snazzy",
     "lint:typescript": "eslint -c types/.eslintrc.json types/**/*.d.ts test/types/**/*.test-d.ts",
-    "unit": "tap -J test/*.test.js test/*/*.test.js",
-    "unit:report": "tap -J test/*.test.js test/*/*.test.js --cov --coverage-report=html --coverage-report=cobertura | tee out.tap",
+    "unit": "npm run gen-version && tap -J test/*.test.js test/*/*.test.js",
+    "unit:report": "npm run gen-version && tap -J test/*.test.js test/*/*.test.js --cov --coverage-report=html --coverage-report=cobertura | tee out.tap",
     "unit:junit": "tap-mocha-reporter xunit < out.tap > test/junit-testresults.xml",
     "test:typescript": "tsd",
     "test:report": "npm run lint && npm run unit:report && npm run test:typescript",
     "test": "npm run lint && npm run unit && npm run test:typescript",
     "coverage": "npm run unit -- --cov --coverage-report=html",
     "test:ci": "npm run lint && npm run unit -- --cov --coverage-report=lcovonly && npm run test:typescript",
-    "bench": "branchcmp -r 2 -g -s \"npm run benchmark\"",
-    "benchmark": "npx concurrently -k -s first \"node ./examples/simple.js\" \"npx autocannon -c 100 -d 5 -p 10 localhost:3000/\"",
-    "license-checker": "license-checker --production --onlyAllow=\"MIT;ISC;BSD-3-Clause;BSD-2-Clause\""
+    "bench": "npm run gen-version && branchcmp -r 2 -g -s \"npm run benchmark\"",
+    "benchmark": "npm run gen-version && npx concurrently -k -s first \"node ./examples/simple.js\" \"npx autocannon -c 100 -d 5 -p 10 localhost:3000/\"",
+    "license-checker": "license-checker --production --onlyAllow=\"MIT;ISC;BSD-3-Clause;BSD-2-Clause\"",
+    "gen-version": "node build/build-version.js"
```

Comment:
Shouldn't we also add this to prepublish hook?

Reply:
I'm not aware of the publish stack. Can add if required.

Reply:
https://github.com/fastify/releasify is usually used.
Since it eventually does `npm publish`, I think standard npm prepublish hooks should work fine. `prepublishOnly` seems like the right one.